### PR TITLE
Bug 1344008 : added variables to the cluster variables section of the advanced install

### PR DESCRIPTION
**Pull request by bfallonf**
Not sure if there's much more to add.

Asking for an ack in the BZ.

### CODE REVIEW

**Comment by bfallonf** [comment]
Ack given.

@tpoitras @ahardin-rh @tnguyen-rh @adellape Available for peer review.

**Comment by bfallonf** [comment]
[rev_history]
|link:../install_config/install/advanced_install.html[Installing -> Advanced Installation]
|Added more variables to the link:../install_config/install/advanced_install.html#configuring-cluster-variables[Configuring Cluster Variables section].
%

**Comment by tnguyen-rh** [comment]
@bfallonf

The "registries" in the name of the vars (and not "registry") seems to indicate that the value of the variable can be any number of hostnames.  If that's the case, what is the delimiter?  Comma?

Also, i'd say "This variable specifies {a host, hosts} to add/reject/..." since the variable itself doesn't do the action.

**Comment by bfallonf** [comment]
@tnguyen-rh Thanks, I made some reviews. Let me know what you think.

As for the comma thing...I guess? I thought it was always commas with these things.

File: install_config/install/advanced_install.adoc [hunk 1]
```diff
@@ -305,6 +305,18 @@ re-configured after deployment.
...
+|`*openshift_docker_additional_registries*`
+|{product-title} adds the specified additional registry or registries to the
+Docker configuration.
+
+|`*openshift_docker_insecure_registries*`
+|{product-title} adds the specified additional insecure registry or registries
+to the Docker configuration.
+
+|`*openshift_docker_blocked_registries*`
+|{product-title} adds the specified rblocked egistry or registries to the Docker
```
Comment (tnguyen-rh) [inline install_config/install/advanced_install.adoc]: typo: "r" too eager :smile:

**Comment by tnguyen-rh** [comment]
@bfallonf 
- I guess if comma is the norm, there's no need to mention it.
- Changes LGTM, bar an errant "r".

**Comment by bfallonf** [comment]
@tnguyen-rh Thanks for the catch! I've updated and will merge when it's built.

**Pull request merged**